app/src/main/res/layout/welcome.xml

```diff
@@ -11,44 +11,46 @@
 
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
-    android:layout_height="wrap_content">
+    android:layout_height="wrap_content"
+    android:paddingStart="28dp"
+    android:paddingEnd="28dp">
 
-    <TextView
-      android:id="@+id/welcome_text_view"
-      android:layout_width="0dp"
+    <com.google.android.flexbox.FlexboxLayout
+      android:id="@+id/flex_layout"
```

Comment:
id naming should be more understandable.

Reply:
Hmm, I forgot to rename it. 
Does `salutation_and_username_flexbox_layout` sounds good?